core-api/src/test/java/com/optimizely/ab/OptimizelyTest.java

```diff
@@ -3495,6 +3496,36 @@ public void isFeatureEnabledWithExperimentKeyForcedOfFeatureEnabledFalse() throw
         assertFalse(optimizely.isFeatureEnabled(FEATURE_FLAG_MULTI_VARIATE_FEATURE.getKey(), testUserId));
     }
 
+    /**
+     * Verify that the {@link Optimizely#activate(String, String, Map<String, String>)} call
+     * uses forced variation to force the user into the second variation in which FeatureEnabled is not set
+     * feature enabled will return false by default
+     */
+    @Test
+    public void isFeatureEnabledWithExperimentKeyForcedWithNoFeatureEnabledSet() throws Exception {
```

Comment:
@thomaszurkan-optimizely  here is the test case I am checking in which we are not setting featureEnabled variable in datafile and in config so its returning false by default